### ISSUE TITLE
NPE when calling MediaType.toString()

### ISSUE THREAD

**Issue by mknudsen** [mrmike/Ok2Curl#1]
I just got this error:

````
10-13 12:08:27.317 1889-1912/? E/AndroidRuntime: FATAL EXCEPTION: OkHttp Dispatcher
10-13 12:08:27.317 1889-1912/? E/AndroidRuntime: Process: my.package.name, PID: 1889
10-13 12:08:27.317 1889-1912/? E/AndroidRuntime: java.lang.NullPointerException: Attempt to invoke virtual method 'java.lang.String com.squareup.okhttp.MediaType.toString()' on a null object reference
10-13 12:08:27.317 1889-1912/? E/AndroidRuntime:     at com.moczul.ok2curl.CurlBuilder.<init>(CurlBuilder.java:35)
10-13 12:08:27.317 1889-1912/? E/AndroidRuntime:     at com.moczul.ok2curl.CurlInterceptor.intercept(CurlInterceptor.java:31)
10-13 12:08:27.317 1889-1912/? E/AndroidRuntime:     at com.squareup.okhttp.Call$ApplicationInterceptorChain.proceed(Call.java:221)
10-13 12:08:27.317 1889-1912/? E/AndroidRuntime:     at com.squareup.okhttp.Call.getResponseWithInterceptorChain(Call.java:195)
10-13 12:08:27.317 1889-1912/? E/AndroidRuntime:     at com.squareup.okhttp.Call.access$100(Call.java:34)
10-13 12:08:27.317 1889-1912/? E/AndroidRuntime:     at com.squareup.okhttp.Call$AsyncCall.execute(Call.java:162)
10-13 12:08:27.317 1889-1912/? E/AndroidRuntime:     at com.squareup.okhttp.internal.NamedRunnable.run(NamedRunnable.java:33)
10-13 12:08:27.317 1889-1912/? E/AndroidRuntime:     at java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1112)
10-13 12:08:27.317 1889-1912/? E/AndroidRuntime:     at java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:587)
10-13 12:08:27.317 1889-1912/? E/AndroidRuntime:     at java.lang.Thread.run(Thread.java:818)
````

I used OkHttp Version 2.5 and Ok2Curl 0.0.2.

**Comment by mrmike** [comment]
Thanks for reporting an issue. I fixed it in version `0.0.3`. Make sure to update version in `build.gradle` file.

Could you paste request which caused this NPE?

Fix:
https://github.com/mrmike/Ok2Curl/commit/22082d3f2deb1f82e4225b92e442bef4dc1ff5d4

**Comment by mknudsen** [comment]
Just some generic request agains one of our backend. Seems fixed now. Thanks!
